OpenSim/Simulation/Model/CMCActuatorSubsystem.cpp

```diff
@@ -146,14 +142,21 @@ void CMCActuatorSubsystemRep::setSpeedTrajectories(FunctionSet *aSet) {
         _qSet->evaluate(_uWork,1,t);
     }
 
-    for(i=0;i<nq;i++) q[i] = _qWork[i] + _qCorrections[i];
-    for(i=0;i<nu;i++) u[i] = _uWork[i] + _uCorrections[i];
+    /* Hack to obtain a modifiable state in a const method */
```

Comment:
Ideally, would `realizeSubsystemDynamicsImpl()` just take a non-const state? The comment seems sufficient if we don't want to start fiddling with CMC's architecture.